contrib/kafka/config/kafka.yaml

```diff
@@ -212,3 +212,32 @@ spec:
         - name: kafka-channel-controller-config
           configMap:
             name: kafka-channel-controller-config
+
+---
+
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
This gives an error when I execute `kubectl apply -f`: error validating "STDIN": error validating data: ValidationError(StatefulSet.spec): missing required field "serviceName" in io.k8s.api.apps.v1.StatefulSetSpec; if you choose to ignore these errors, turn validation off with --validate=false.

Is it an error that we can ignore?